### DISCUSSION TITLE
Welcome to nuxui Discussions!

### DISCUSSION THREAD

**Discussion by nuxart** [nuxui/nuxui#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ahm3dRN** [comment]
This is really a great tool I'm looking forward to use it for production apps 
I'm also hoping a better docs would be here soon so more people can use it and share it with others

If one want to help with docs or something, can we make pull requests this early?
And thanks  a lot for this.

**Comment by nuxart** [comment]
@Ahm3dRN That's cool! Thank you so much!

NuxUI now is in developing, the sdk structure and API maybe changed later.

We used a lot cgo code at render, it very time consuming, the whole widget structure need redesign.

So, it is not a good time for complete docs now, we can do this after frist stable base version .

**Reply by ddkwork** [reply]
Please support table,list,and tree, thank you.

**Reply by nuxart** [reply]
@ddkwork Sure

**Reply by jbhvip** [reply]
输入框现在还没有开发出来吗

**Comment by aasish-go** [comment]
its awesome , need more widget and the widget properties like flutter.
